### DISCUSSION TITLE
middleware/mysql/user-guide/logs/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1915] [Announcements]
# middleware/mysql/user-guide/logs/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/mysql/user-guide/logs/

**Comment by liyuerich** [comment]
实例概览， 配置参数， 备份管理的介绍能加上吗？

**Reply by windsonsea** [reply]
Added in PR #5322
